kong/init.lua

```diff
@@ -219,10 +214,10 @@ local function load_declarative_config(kong_config, entities)
   end
 
   if not kong_config.declarative_config then
-    -- no configuration yet, just build empty plugins plan
-    local ok, err = runloop.build_plugins_plan(utils.uuid())
+    -- no configuration yet, just build empty plugins iterator
+    local ok, err = runloop.build_plugins_iterator(utils.uuid())
```

Comment:
```suggestion
    local ok, err = runloop.build_plugins_iterator("init")
```

Reply:
I tried that change in isolation here, but some declarative tests failed: https://travis-ci.org/Kong/kong/builds/530721676 (the branch is erased now, but it only contained that change - "init" instead of `utils.uuid()`)

Reply:
yes, those issues where all fixed the on async branch

Reply:
if i remember correctly the declarative uses the uuid to determine later on that it has changed and will start to build again. I can check this on monday.

Reply:
Ok, you refer to this (so let it be like it is then):
https://github.com/Kong/kong/pull/4595